### DISCUSSION TITLE
How do I tell if Anti-kick is enabled?.

### DISCUSSION THREAD

**Discussion by joep26020** [winstxnhdw/lc-hax#139] [Q&A]
### What happened?

Also I want to say I love the work and I'm willing to donate money for this cause because how great of a dev you are Winston!

### Current Commit Hash

515515515

### Injector

- [ ] SharpMonoInjectorCore
- [ ] Others

### Acknowledgement

- [X] I have **recursively** cloned the repository as instructed in the [README](https://github.com/winstxnhdw/lc-hax/edit/main/README.md)
- [X] I have done my due diligence to check for similar issues

**Comment by winstxnhdw** [comment]
There's a label at the top right while you are in the menu screen that will tell you.
